js/tests/components/song/list.spec.js

```diff
@@ -136,4 +138,128 @@ describe('components/song/list', () => {
     wrapper.find('.song-list-wrap').trigger('keydown.a', { ctrlKey: true })
     wrapper.vm.filteredItems.forEach(item => expect(item.selected).toBe(true))
   })
+
+  describe(`mounted()`, () => {
+    let orderBy
+    beforeEach(() => {
+      orderBy = jest.spyOn(filters, "orderBy")
+    })
+
+    it(`when sorting is disallowed, it will not sort`, () => {
```

Comment:
Same as other comments. Also please use present tense e.g. `it('does not sort when…')`